app/app/settings.py

```diff
@@ -939,3 +939,6 @@ def callback(request):
 # GTC Token Distribution
 GTC_DIST_API_URL = env('GTC_DIST_API_URL', default='http://localhost:8000/not-valid-url')
 GTC_DIST_KEY = env('GTC_DIST_KEY', default='')
+DIDKIT_KEY_JWK = env.json('DIDKIT_KEY_JWK', default={})
+POPP_VC_ISSUER = env.str('POPP_VC_ISSUER', default='')
+POPP_VC_VERIFIER = env.str('POPP_VC_VERIFIER', default='')
```

Comment:
^ could we add in doc on where this would be populated from

Reply:
Added in https://github.com/gitcoinco/web/pull/9032/commits/0523e1bae66c9b948c3710972394d3085d220310